app/src/renderer/components/governance/TableProposals.vue

```diff
@@ -94,6 +94,12 @@ export default {
           tooltip: `The title of the proposal`,
           class: `proposal_title`
         },
+        {
+          title: `Submitted Block`,
```

Comment:
sorting by proposal id is the same

Reply:
true, but i added a column for submit block instead. do you think showing ID as its own column would be better?

Reply:
we can discuss it later